presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestBaseJdbcConfig.java

```diff
@@ -67,6 +68,7 @@ public void testExplicitPropertyMappings()
 
         assertFullMapping(properties, expected);
 
+        assertEquals(expected.getJdbcTypesMappedToVarchar(), ImmutableSet.of("mytype", "struct_type1"));
```

Comment:
You can now remove `assertEquals(expected.getJdbcTypesMappedToVarchar().contains ...` lines